torchvision/prototype/transforms/_augment.py

```diff
@@ -7,15 +7,15 @@
 import torch
 from torch.utils._pytree import tree_flatten, tree_unflatten
 from torchvision.ops import masks_to_boxes
-from torchvision.prototype import features
+from torchvision.prototype import datapoints
 from torchvision.prototype.transforms import functional as F, InterpolationMode
 
 from ._transform import _RandomApplyTransform
 from .utils import has_any, query_chw, query_spatial_size
 
 
 class RandomErasing(_RandomApplyTransform):
-    _transformed_types = (features.is_simple_tensor, features.Image, PIL.Image.Image, features.Video)
+    _transformed_types = (datapoints.is_simple_tensor, datapoints.Image, PIL.Image.Image, datapoints.Video)
```

Comment:
Driveby comment:
I remember that we placed `is_simple_tensor()` next to Feature because back then we didn't have a utils file. Does it make sense to move it now in utils on a follow up PR?

Reply:
IIRC, there was some other cyclic dependency issue with it. Let me try to see if I can find the reason and whether it is still valid. If not, we can move it to public `utils`.

Reply:
We are using it in `transforms.functional`

https://github.com/pytorch/vision/blob/790f1cdcea0359619adfc9ec37b91883748d1854/torchvision/prototype/transforms/functional/_misc.py#L57

but we can avoid the cyclic dependency.